qualtricssurvey/templates/view.html

```diff
@@ -2,7 +2,7 @@
     <div class="qualtrics_message"> 
         <div class="grade">
           <span class="earned_score">{{earned_score}}</span>/<span class="max_score">{{max_score}}</span> points 
-          <span class="status"></span>
+          <span class = "is_graded"></span><span class="status"></span>
```

Comment:
@bsande6 Can you remove the spaces around the `=` sign for the `class` attribute?